stock_financial_risk/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,10 @@
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+  * Carlos Dauden
+  * Pedro M. Baeza
+  * David Vidal
+  * Ernesto Tejeda
+
+* `Jarsa <https://www.jarsa.com.mx>`_:
+
+  * Alan Ramos
```

Comment:
If this will be contributed back to OCA, perhaps you ought to include yourself as contributor?